src/scanner/arp.go

```diff
@@ -176,10 +176,11 @@ func (a *ArpScanner) readARP(handle *pcap.Handle, iface *net.Interface) {
 
 			a.logger.Debugf("ARP Packet received for [MAC: %v] [IP: %v]", net.HardwareAddr(arp.SourceHwAddress), net.IP(arp.SourceProtAddress))
 			tmpMacAddress := net.HardwareAddr(arp.SourceHwAddress).String()
+			ip := net.IP(arp.SourceProtAddress).String()
 
 			newDevice := model.NewDevice{
 				Mac:  &tmpMacAddress,
-				Host: net.IP(arp.SourceProtAddress).String(),
+				Host: ip,
```

Comment:
could maybe revert? or are you looking for more readability here?

Reply:
I changed it because I was playing with type and ended up leaving it because it's a little more legible